packages/api/src/models/Bills.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import Money, { transformMoneyRequest, transformMoneyResponse } from './Money'
+import { DateFormat, transformDateRequest, transformDateResponse } from './Date'
+import { ErrorResponse, isAccountingErrorResponse, transformErrorResponse } from './Error'
+import Pagination from './Pagination'
+import BillPayments, { transformBillPaymentsData } from './BillPayments'
+import VisState from './VisState'
+import BillLines, { transformBillLinesRequest, transformBillLinesResponse } from './BillLines'
+
+enum BillStatus {
+	unpaid = 'unpaid',
+	overdue = 'overdue',
+	partial = 'partial',
+	paid = 'paid',
+}
+
+export default interface Bills {
+	id?: number
+	amount?: Money
+	attachment?: object
+	billNumber?: string
+	billPayments?: BillPayments[]
+	createdAt?: Date
+	currencyCode?: string
+	dueDate?: Date
+	dueOffsetDays?: number
+	issueDate?: Date
+	language?: string
+	lines?: BillLines[]
+	outstanding?: Money
+	overallCategory?: string
+	overallDescription?: string
+	paid?: Money
+	status?: BillStatus
+	taxAmount?: Money
+	totalAmount?: Money
+	updatedAt?: Date
+	vendorId: number
+	visState?: VisState
+	vendor?: object
+}
+
+export function transformBillsData(bill: any): Bills {
+	return {
+		id: bill.id,
+		amount: bill.amount && transformMoneyResponse(bill.amount),
+		attachment: bill.attachment,
+		billNumber: bill.bill_number,
+		billPayments:
+			bill.bill_payments &&
+			bill.bill_payments.map((billPayment: any): BillPayments => transformBillPaymentsData(billPayment)),
+		createdAt: bill.created_at && transformDateResponse(bill.created_at, DateFormat['YYYY-MM-DD hh:mm:ss']),
```

Comment:
Sadly, most of the dates from the accounting endpoints return in `America/New_York` time. However these new endpoints are in UTC. The transformDateResponse with this format will assign this the wrong timezone (see https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/pin-axios/packages/api/src/models/Date.ts#L20)

For the bill created_at and updated_at fields, we should either do something like this: https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/master/packages/api/src/models/Client.ts#L96 (another field in UTC), or maybe add another enum value here (https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/pin-axios/packages/api/src/models/Date.ts#L3) like `YYYY-MM-DD hh:mm:ss UTC` and handle it properly here https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/pin-axios/packages/api/src/models/Date.ts#L19-L21 with `zone: 'UTC'`.